app/models/discussion_reader.rb

```diff
@@ -67,6 +67,10 @@ def discussion_reader_volume
     self.class.volumes.invert[self[:volume]]
   end
 
+  def sync_read_items_count!
+    update_attribute(:read_items_count, calculate_read_items_count)
```

Comment:
I can't see any place in the server side where we use this anymore.

The client calculates it's own read items count pretty quickly anyway. I think we can drop the job ( and column later, with the other columns I'm yet to drop).